### ISSUE TITLE
Remove obsolete cve_scanner package

### ISSUE THREAD

**Issue by mpreisler** [OpenSCAP/openscap-daemon#93]
CVE scanning is now done as a task, the cve_scanner package is of no use. Let's remove it so that we don't have to maintain it any more.

**Comment by jan-cerny** [comment]
I don't think we can remove it, because `oscapd-cli` scan uses `cli_scan` which uses `CVEScanListAsync` which uses `Worker` which is defined in cve_scanner package.
@dahaic @mpreisler @matejak @yuumasato @mildass @matusmarhefka  Do we want to drop this functionality from `oscapd-cli`?

**Comment by dahaic** [comment]
@matusmarhefka `atomic scan` is just using daemon. Let's look at it from the perspective of daemon, not one particular use case.

@jan-cerny I would suggest to preserve the functionality.

**Comment by jan-cerny** [comment]
@matusmarhefka I think it isn't using `oscapd-cli`. I think you're right that it uses `oscapd-evaluate`.

**Comment by jan-cerny** [comment]
@dahaic I agree that we should preserve the functionality.

**Comment by jan-cerny** [comment]
Do you think we should rework the code to avoid using the `cve_scanner` package?

**Comment by mpreisler** [comment]
the cli_scan stuff is virtually unused, it was replaced with oscapd-evaluate and the task API VulnerabilityScan mode. We can remove cli_scan IMO, it's obsolete anyway.
